website/docs/quickstart.md

```diff
@@ -117,17 +122,31 @@ The Subo REPL includes a proxy that makes it easy to connect the hosted editor t
 
 The `local.suborbital.network` subdomain points to `127.0.0.1`, i.e. `localhost`. You may need to substitute a different hostname or IP address depending on your particular network setup.
 
+## Create a tenant (user)
+
+Suborbital lets an application's users create their own secure, sandboxed plugins, carefully isolated from the core of the system and one another. For this reason, we will create a new tenant, which is a user account with its own plugins inside Suborbital. Our application will then connect the tenant with one of its own internally maintained users.
+
+Run this command with changes listed below it:
+
+```bash
+curl --location --request POST "https://controlplane.stg.suborbital.network/api/v2/tenant/${IDENTIFIER}" \
```

Comment:
Looks a staging URL slipped in here

Reply:
Also gone due to not needing this section.